client/src/views/client-area/personal-data/PersonalInfoModal.vue

```diff
@@ -94,7 +95,7 @@ async function submit(): Promise<boolean> {
     errors.value = response.errors ?? [];
     return false;
   }
-  return await modalController.dismiss({}, MsModalResult.Confirm);
+  return await modalController.dismiss(`${translationPrefix}.success`, MsModalResult.Confirm);
```

Comment:
The modal should return a status, and this status should be used by the calling component to show a message, it should not return the message directly.

Reply:
done